jormungandr/src/main.rs

```diff
@@ -74,9 +75,10 @@ pub struct BootstrappedNode {
 
 const BLOCK_TASK_QUEUE_LEN: usize = 32;
 const FRAGMENT_TASK_QUEUE_LEN: usize = 1024;
-const NETWORK_TASK_QUEUE_LEN: usize = 32;
+const NETWORK_TASK_QUEUE_LEN: usize = 64;
```

Comment:
Any observations that caused you to increase this?

Reply:
We now send gossips to the network task as well (in addition to whatever was sent before) and they could be ~30 messages each gossip interval. I also put this in the commit message.

Reply:
Yeah, saw the comment in https://github.com/input-output-hk/jormungandr/pull/3177/commits/1e92b76c39e0417c6e868b9806afd7328e6e1b2f after I posted this.